node/src/Pos/Client/Txp/Util.hs

```diff
@@ -257,16 +262,40 @@ makeLenses ''InputPickerState
 
 type InputPicker = StateT InputPickerState (Either TxError)
 
+-- | Filters the input '[PendingTx]' to choose only the ones which are not
+-- yet persisted in the blockchain.
+nonConfirmedTransactions :: [PendingTx] -> [PendingTx]
```

Comment:
Btw some minor comment: this module lies in `cardano-sl` subproject and contains general logic for transaction creation used all over the project, while `PendingTx` is purely wallet's stuff.
And this is actually going to fire, because module, which declares `PendingTx` and related stuff and yet lies in `cardano-sl`, has already been moved to `cardano-sl-wallet` in `master`, and now merge to `master` promises to be slightly painful :slightly_smiling_face: 

So it would be very nice to move these two functions to, say, `Pos.Wallet.Web.Pending.Util`, and here pass `[Address]` around.

Reply:
I'm happy to go with the most pragmatic approach -- I don't mind backporting manually the policy on `master` and `cardano-sl-1.0` rather than cherry-picking, if that means shipping the fix quicker 😉